tests/__tests__/disable-sourcemap-support.spec.ts

```diff
@@ -8,7 +8,8 @@ describe('Typescript errors', () => {
     const stderr = result.stderr;
 
     // The actual error is on line 18 - the line# being wrong here means sourcemaps are disabled.
-    expect(stderr).toContain('Hello.ts:8');
+    expect(stderr).toMatch(/at .+Hello\.ts:\d+/);
```

Comment:
This doesn't look right. Don't we want to check for exact line numbers to ensure that the sourcemaps are working correctly?

Reply:
it looks weird, I guess we should change the test description/comments. But here we are testing that, **without** sourcemaps, the line number reported is NOT the correct one, ie there is an error, but NOT on line 18. That is why I put 2 tests, first that there is an error at `Hello.ts` on some line, and then that the line number is NOT 18.

Reply:
https://github.com/huafu/ts-jest/blob/fb5cd12a664b159ac841dfa3646f766a75f84134/tests/__tests__/disable-sourcemap-support.spec.ts#L10